### PR TITLE
fixes the TLS doc, per app

### PR DESCRIPTION
**Pull request by sglebs**
Unfortunately this pull request also includes a change I did for vmWare. I killed my original fork, created a new one but that change survived (I guess GitHub reuses it). Feel free to get just the doc change and ignore the vmware part.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -30,6 +30,11 @@ Vagrant::configure("2") do |config|
     vb.customize ["modifyvm", :id, "--memory", BOX_MEMORY]
   end
 
+  config.vm.provider :vmware_fusion do |vbox, override|
+    vbox.memory = 1024
+    override.vm.box = "phusion/ubuntu-14.04-amd64"
```
Comment (josegonzalez) [inline Vagrantfile]: Just curious, what was the benefit of using this box?

Comment (michaelshobbs) [inline Vagrantfile]: @sglebs did you intent to include the Vagrantfile in this PR? Seems non-sequitur to the PR title...

Comment (josegonzalez) [inline Vagrantfile]: > Feel free to get just the doc change and ignore the vmware part.

**Comment by michaelshobbs** [comment]
I can't read :disappointed:

**Comment by josegonzalez** [comment]
Closed by #1250. Thanks @sglebs!

**Pull request closed without merging**